metaci/release/models.py

```diff
@@ -85,7 +85,7 @@ def get_production_date():
 
     class Meta:
         get_latest_by = "production_push_date"
-        ordering = ["-production_push_date"]
+        ordering = ["-git_tag"]
```

Comment:
This won't work right -- the sorting for a text field is alphabetical, so a tag like release/9 will sort before release/10 in descending order.

If the goal is to show the most recently created release first, I'd suggest `-created`

Reply:
You'll also need to add a migration.